### DISCUSSION TITLE
Hydration failed when using rewrite

### DISCUSSION THREAD

**Discussion by dottodot** [vercel/next.js#48649] [Help]
### Summary

I've got a strange issue when trying to rewrite a url i.e

```
  async rewrites() {
    return {
      beforeFiles: [
        {
          source: '/',
          destination: '/night',
        }
      ],
    };
  },
```
where the page loads but I get the error of 

Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.

Warning: Did not expect server HTML to contain a `<div>` in `<div>`.

If i access the /night url directly it's fine, so a bit stumped as to what would be causing this.

**Comment by louishulot** [comment]
Any news on this one ?
